Policies/github-enterprise-agreement.md

```diff
@@ -152,23 +152,21 @@ If you choose to use GitHub Enterprise Server, the following terms will apply to
 
 "Update" means a Software release that GitHub makes generally available to its customers, along with any corresponding changes to Documentation, that is comprised of an error correction or bug fix, generally indicated by a change in the digit to the right of the second decimal point (e.g., x.x.x to x.x.y).
 
-"User" means a single person or machine account that initiates the execution of the Software or interacts with or directs the Software in the performance of its functions. The number of Users should not exceed the number of Seats that you have licensed.
+"User" means a single person or machine account that initiates the execution of the Software or interacts with or directs the Software in the performance of its functions. 
 
-#### 2. License Grant
+#### 2. Enterprise Server Software License Grant
 
-Subject to your compliance with this Agreement, GitHub grants to you a non-exclusive, non-transferable, worldwide, royalty-free, limited-term license to install and use a single production License of the Software (unless otherwise stated in an Order Form) for your internal business purposes during the applicable Subscription Term, in accordance with the Documentation, and only for the number and type of Seats stated in your Order Form. You may make copies of the Software for non-production purposes only, such as testing, staging or quality assurance purposes, *provided* that you reproduce all copyright and other proprietary notices that appear on the original copy of the Software. Your Affiliates, agents and contractors (collectively, "Authorized Third Parties") may also use the Software, so long as they are using it on your behalf and you remain fully responsible for such Authorized Third Parties' behavior under this Agreement.
+Subject to your compliance with this Agreement, GitHub grants to you a non-exclusive, non-transferable, worldwide, royalty-free, limited-term license to install and use the Software (unless otherwise stated in an Order Form) for your internal business purposes during the applicable Subscription Term, in accordance with the Documentation, and only for the number of Subscription Licenses stated in your Order Form. You may make copies of the Software for non-production purposes only, such as testing, staging or quality assurance purposes, *provided* that you reproduce all copyright and other proprietary notices that appear on the original copy of the Software. Your Affiliates, agents and contractors (collectively, "Authorized Third Parties") may also use the Software, so long as additional Subscription Licenses have been purchased for their Users, they are using the Software on your behalf, and you remain fully responsible for such Authorized Third Parties' behavior under this Agreement.
```

Comment:
@meentastic per our team meeting today, there is no longer a restriction on the number of production instances that a Customer can have. So I can't think of a reason we need to distinguish between the number of production instances and the number of non-production copies. So wondering if can we remove the sentence altogether that references using copies of the software for non-production purposes? 

~~You may make copies of the Software for non-production purposes only, such as testing, staging or quality assurance purposes, *provided* that you reproduce all copyright and other proprietary notices that appear on the original copy of the Software.~~

cc/ @lyhashim @dapont @theseawitch